crates/red_knot_python_semantic/build.rs

```diff
@@ -0,0 +1,4 @@
+/// Rebuild the crate if a test file is added or removed from
+pub fn main() {
+    println!("cargo:rerun-if-changed=resources/mdtest/*.md");
```

Comment:
I ran into #13732 as well this morning, so I appreciate that we try to fix it. But it looks like doing this with a glob forces a recompilation *every time*, even if nothing has changed in the Markdown files(?).

When working on these tests, most of the time we're probably just editing Markdown files, not adding new ones. With this change, we make all of of those `cargo test`/`cargo test mdtest`/… calls much slower. We can probably workaround this locally by running the test executable directly (not via `cargo`). But that's another thing that developers would need to be taught :face_with_diagonal_mouth:

Reply:
> It looks like maybe we're not really meant to use globs here

Ah, yes. That was the problem. I can confirm that it works as intended now :+1: 

> I think the slowdown in having to rebuild the crate to run the tests is a _reasonable_ price to pay for avoiding the confusion from #13732

Agreed, #13732 should be fixed. And I don't have any other ideas on how to solve it... except for a hard-coded list of Markdown files somewhere, and that would be a big downside in terms of ergonomics.